std/array.d

```diff
@@ -2652,8 +2661,18 @@ if (isDynamicArray!(E[]) &&
 }
 
 /++
-    Returns a new array that is $(D s) with $(D slice) replaced by
-    $(D replacement[]).
+    Creates a new array such that the items in `slice` are replace with the
+    items in `replacement`. `slice` and replacement do not need to be the
```

Comment:
For consistency I would put the second occurrence of replacement in backticks as well.
Moreover I think a `length` is a property, so a verb denominating ownership like "have" or "share" would be more appropriate.